### ISSUE TITLE
Fittings AJAX call returns a 500 Error if the ship type no longer exists

### ISSUE THREAD

**Issue by OperatorOverload** [eveseat/seat#735]
TL;DR: If a character has a fitting in EVE for a ship that no longer exists in game (and isn't in the SDE), the fitting paging fails with a 500 error when you try to page to the data table page that would contain that ship.

Background: Over the course of EVE history, a number of ships have existed that have since been removed.  These were typically Dev ships or one-time use ships that have been phased out, and have since had their types removed from the SDE.  Interestingly enough, these fittings are still in-game, even thought they're no longer valid, and are returned by the ESI when you pull fittings. This is crazy on CCP's part, the fittings should be purged when the ship is removed, but for whatever reason, they have not been.  There are several of these, but for the sake of our case here, the one in question is the Wraith which was removed in the Rubicon 1.3 / March 2014 release: https://wiki.eveuniversity.org/Wraith

When trying to load fittings for a character that has that fit saved, the Fittings data table for the page that it's on fails to load.  Other pages for the same character load fine, but anything that is trying to get to that fitting doesn't load correctly.  The top of the call stack is as follows:
```
[2020-12-28 21:49:19] local.ERROR: Trying to get property 'price' of non-object {"userId":1,"exception":"[object] (Yajra\\DataTables\\Exceptions\\Exception(code: 0): Trying to get property 'price' of non-object at /var/www/seat/vendor/yajra/laravel-datatables-oracle/src/DataTableAbstract.php:862)
[stacktrace]
#0 /var/www/seat/vendor/yajra/laravel-datatables-oracle/src/QueryDataTable.php(106): Yajra\\DataTables\\DataTableAbstract->errorResponse(Object(ErrorException))
#1 /var/www/seat/vendor/eveseat/web/src/Http/DataTables/Common/Military/AbstractFittingDataTable.php(85): Yajra\\DataTables\\QueryDataTable->make(true)
#2 /var/www/seat/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Seat\\Web\\Http\\DataTables\\Common\\Military\\AbstractFittingDataTable->ajax()
```

The ship typeId is 610, which no longer exists in the SDE, but I dug out an old backup and confirmed it was indeed the Wraith.  Here is the relevant sections of the SeAT tables:
![image](https://user-images.githubusercontent.com/4984990/103248069-c3f70080-492e-11eb-863b-27556fbd2d7c.png)

IMO, SeAT should gracefully fail if a ship type doesn't exist, either ignoring it and not including it in the list (probably not optimal as hard to debug problems), or replacing it with a note that the Type isn't available, but providing other available information in case there are future bugs that cause ship types to be missing unexpectedly.
